1. Добавлена таблица для хранения комментариев

BorodaPT opened the pull request:
2. Добавлена настройка hibernate дла форматирования запросов в schema.sql
3. Добавлен контроллер для эндпойнтов комментариев
4. Добавлен сервис по обработке комментариев
5. Текущая функциональность поддерживает:
- CRUD операции для залогированных пользователей
- RUD операции для администратора
- дополнительно администратор имеет право устанавливать "видимость" комментариев

BorodaPT committed:
1. Добавлена таблица для хранения комментариев
2. Добавлена настройка hibernate дла форматирования запросов в schema.sql
3. Добавлен контроллер для эндпойнтов комментариев
4. Добавлен сервис по обработке комментариев
5. Текущая функциональность поддерживает:
- CRUD операции для залогированных пользователей
- RUD операции для администратора
- дополнительно администратор имеет право устанавливать "видимость" комментариев

BorodaPT committed:
checkStyle

BorodaPT committed:
refactor:
- подгон тестовых значений пользователя и события под пре-тест значения

BorodaPT committed:
refactor:
README.md

db-exp commented on main_ewm/main_ewm_service/src/main/java/ru/practicum/services/comments/ServiceCommentImpl.java:
```diff
@@ -0,0 +1,152 @@
+import java.util.List;
+
+@Service
+@Transactional(readOnly = true)
```
в интерфейсе можно не проставлять и оставить только в реализации

db-exp commented on main_ewm/main_ewm_service/src/main/java/ru/practicum/services/comments/ServiceCommentImpl.java:
```diff
@@ -0,0 +1,152 @@
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class ServiceCommentImpl implements ServiceComment {
```
правильней называть CommentServiceImpl  - общий паттерн Сущность(Сервис/Контроллер)

db-exp commented on main_ewm/main_ewm_service/src/main/resources/schema.sql:
Молодец!

db-exp commented on main_ewm/main_ewm_service/src/main/resources/application.properties:
```diff
@@ -4,6 +4,7 @@ stat-server.url=http://stats-server:9090
 spring.jpa.hibernate.ddl-auto=none
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL10Dialect
 spring.jpa.properties.hibernate.format_sql=true
```
просто заметка - такие вещи хорошо использовать только для отладки и выключать при выпуске в прод

BorodaPT committed:
refactor:
1. Убран лишний код в соотвтетсвии с комментариями;
2. Наименования классов приведены к соотвтетствию паттерну "наименований";

db-exp reviewed (approved):
Огонь!
